modules/sage/1.0/config/default.yaml

```diff
@@ -0,0 +1,43 @@
+lcr-modules:
+    
+    sage:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            
+
+        scratch_subdirectories: []
+
+        # include here any additional flags to modify default parameters
+        options:
+            sage_run: ""
+
+
+        conda_envs:
+            sage: "{MODSDIR}/envs/hmftools-sage-2.2.yaml"
+            bcftools: "{MODSDIR}/envs/bcftools-1.10.2.yaml"
+            
+        resources:
+            sage_run:
+                mem_mb: 60000
+                disk_mb: 200000 # Arbitrary number to ensure at least this much space is left
+                bam: 1
+            filter:
+                mem_mb: 8000
+                disk_mb: 200000 # Arbitrary number to ensure at least this much space is left
+        
+        threads:
+            sage_run: 24
+            filter: 1
+
+
+        pairing_config:
+            genome:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
```

Comment:
I agree with this implementation but I am wondering if we should also support running without a normal if that option is available. This could also be added to a later version.